source/geod24/LocalRest.d

```diff
@@ -48,6 +48,10 @@ private struct Command
 {
     /// Tid of the sender thread (cannot be JSON serialized)
     C.Tid sender;
+    /// In order to support re-entrancy, every request contains an id
+    /// which should be copied in the `Response`
+    /// Initialized to `size_t.max` so not setting it crashes the program
+    size_t id = size_t.max;
```

Comment:
Where is the code that crashes it if it's equal to size_t.max?

Reply:
The array indexing here: https://github.com/Geod24/localrest/pull/4/files#diff-b5229c55d46fb0a852c49674585a3595R311